### PR TITLE
[stable/luigi] support pod disruption budget in chart

### LINKED ISSUE #13149: [stable/luigi] support pod disruption budget in chart
**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

In my use case, the we occasionally need to cycle/drain nodes in the cluster, thus to ensure high availability of Lugi service in those disruption times, wonder if we can support PodDisruptionBudget in chart.

**Version of Helm and Kubernetes**: Helm(Server) v2.12.1 (Client) v2.11.0; kubernetes Major:"1", Minor:"11+"

**Which chart**: stable/lugi

**What you expected to happen**: support a minimum of pods working when disruption happens

**Anything else we need to know**: for reference - detail for pdb https://kubernetes.io/docs/tasks/run-application/configure-pdb/ and disruption: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/

### PR DESCRIPTION
**Pull request by zluo-rmn**
Signed-off-by: Zhongyi Luo <zluo@rmn.com>

#### What this PR does / why we need it:
When there's need to cycle/drain nodes in the cluster, to ensure high availability of Lugi service in those disruption times, we can support PodDisruptionBudget in chart. for more information: https://kubernetes.io/docs/tasks/run-application/configure-pdb/

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #13149

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/luigi/values.yaml [hunk 1]
```diff
@@ -83,3 +83,7 @@ mysql:
   mysqlAllowEmptyPassword: true
   persistence:
     enabled: false
+
+podDisruptionBudget:
+  enabled: true
```
Comment (cpanato) [inline stable/luigi/values.yaml]: I would say this should be false by default

Comment (zluo-rmn) [reply stable/luigi/values.yaml]: thanks, I've pushed a corresponding change with squashed commit.

File: stable/luigi/Chart.yaml [hunk 2]
```diff
@@ -18,5 +18,5 @@ keywords:
 - spark
 - kubernetes job manager
 name: luigi
-version: 2.7.4
+version: 2.7.5
```
Comment (davidkarlsen) [inline stable/luigi/Chart.yaml]:
```suggestion
version: 2.8.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/luigi/Chart.yaml (hunk 2 above)
Comment (zluo-rmn) [reply stable/luigi/Chart.yaml]: Thanks, changed to bump minor version in current commit (with squash)

**Comment by victoriaalee** [comment]
@cpanato @davidkarlsen Could this PR be reopened? The original contributor has since left our company, but we are picking up this work again.

**Pull request closed without merging**